h2o-docs/src/product/data-science/algo-params/monotone_constraints.rst

```diff
@@ -0,0 +1,52 @@
+``monotone_constraints``
+------------------------
+
+- Available in: GBM, XGBoost
+- Hyperparameter: no
+
+Description
+~~~~~~~~~~~
+
+A mapping that represents monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint. Note that constraints can only be defined for numerical columns. 
+
+**Note**: This option can only be used when the distribution is either ``gaussian`` or ``bernoulli``.
+
+Related Parameters
+~~~~~~~~~~~~~~~~~~
+
+- None
+
+Example
+~~~~~~~
+
+.. example-code::
+   .. code-block:: r
+
+	library(h2o)
+	h2o.init()
+
+	#import the prostate dataset:
+	prostate = h2o.importFile("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate.csv.zip")
+	#wip
+	prostate$CAPSULE <- as.factor(prostate$CAPSULE)
+	response <- "CAPSULE"
+	#enforce a constraint
```

Comment:
Add a space before the following comment:
"# train a model using the monotone_constraints option"